Code/Tools/AssetProcessor/native/ui/BuilderDataItem.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AZCore/std/containers/vector.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/smart_ptr/shared_ptr.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <AzCore/std/string/string.h>
+
+namespace AssetProcessor
+{
+    class BuilderDataItem
+    {
+    public:
+        enum class JobType
+        {
+            CreateJob,
```

Comment:
is this supposed to be CreateJobs instead of CreateJob?

Reply:
I think it is CreateJob but now I am not sure how does people use the term.

In my viewpoint, the combination of a source asset and a builder uniquely identifies one CreateJob. Previously in Asset Processor Manager, the signal CreateJobsDurationChanged (assetProcessorManager.h:263) is emitted when *all builders* interested in the source asset finish their CreateJobs. Since there can be multiple builders acting on the source asset, there could be multiple CreateJobs, thus the signal name takes plural form. In this case, I am specifying the *type* of the job, which can be either CreateJob or ProcessJob, I think the singular form is appropriate.

Probably I am understanding the term incorrectly again?! It came up to me that there was only a single kind of job, and a single CreateJobs process could possibly "create" multiple "jobs", thus the name "CreateJobs"? In that case, CreateJobs sounds more reasonable.

Reply:
Resolved: changed to CreateJobs